storage/src/vespa/storage/bucketdb/bucketmanager.cpp

```diff
@@ -305,10 +305,9 @@ void BucketManager::run(framework::ThreadHandle& thread)
                 thread.registerTick(framework::PROCESS_CYCLE);
             }
         }
-        if (timeToCheckMinUsedBits < _component.getClock().getTimeInMillis()) {
+        if (timeToCheckMinUsedBits < _component.getClock().getMonotonicTime()) {
             updateMinUsedBits();
-            timeToCheckMinUsedBits = _component.getClock().getTimeInMillis();
-            timeToCheckMinUsedBits += framework::MilliSecTime(CHECK_MINUSEDBITS_INTERVAL);
+            timeToCheckMinUsedBits = _component.getClock().getMonotonicTime() + CHECK_MINUSEDBITS_INTERVAL;
```

Comment:
Should we change this to only read the clock once?

Reply:
As long as duration of updateMinUsedBits is sane (much less than CHECK_MINUSEDBITS_INTERVAL).